app/kubemci/cmd/delete.go

```diff
@@ -52,6 +52,9 @@ type DeleteOptions struct {
 	// Required
 	// TODO(nikhiljindal): This should be optional. Figure it out from gcloud settings.
 	GCPProject string
+	// Name of the namespace for the ingress. Overrides the namespace in the IngressFilename (if present).
+	// Optional.
+	namespace string
```

Comment:
This one field being private seems odd.
I assume we can change all the fields here and make them private?

Reply:
moved to Namespace, you can privatize them all later.